ocaml/idl/datamodel.ml

```diff
@@ -1515,7 +1515,9 @@ let _ =
   error Api_errors.pusb_vdi_conflict [ "PUSB"; "VDI" ]
     ~doc:"The VDI corresponding to this PUSB has existing VBDs." ();
   error Api_errors.vm_has_vusbs ["VM"]
-    ~doc:"The operation is not allowed when the VM has VUSBs." ()
+    ~doc:"The operation is not allowed when the VM has VUSBs." ();
+  error Api_errors.usb_vm_on_different_host [ "host" ]
+    ~doc:"USB and VM resident on different host." ()
```

Comment:
I would suggest to reword this message a little bit. 
1. USB is quite general, make it more specific such as "The USB device being pass-through";
2. Similar issue as VM;
3. The description stay quite on the reason side of the problem so to reader might get lost as what to do, it is better to describe the requirement more directly, such as:
The USB device being pass-through and the VM to host it have to be on the same host.

Reply:
good suggestion on the message

Reply:
BTW, how about pusb_not_in_possible_host? I think this may be better

Reply:
I am good with it.